src/main/java/core/mate/academy/model/Truck.java

```diff
@@ -5,9 +5,35 @@
  * Do not remove no-field constructor
  */
 public class Truck extends Machine {
+    private int cargoCapacity;
+    private int numberOfAxles;
```

Comment:
The fields `cargoCapacity` and `numberOfAxles` should have access modifiers to encapsulate the properties of `Truck`. Consider using `private` to restrict access to these fields and provide getters and setters for access.